### PR TITLE
add sklearn version to external version in sklearn flows,

### LINKED ISSUE #734: Still not able to use non-sklearn estimators without wrapping them in a pipeline
Hi there, I raised this in issue #724. I have been trying to run experiments with a fairly new sklearn-extra classifier ([https://github.com/Alex7Li/scikit-learn-extra/tree/master/sklearn_extra](url)). The classifier runs fine on a local dataset. However, when I am trying to run it on an openml task, I am getting an error.

Here is a minimal example: 
```python
# define classifier
from sklearn_extra.fast_kernel import FKC_EigenPro
clf = FKC_EigenPro()
# get task
task = openml.tasks.get_task(3)
# run model on task
run = openml.runs.run_model_on_task(clf, task)
# publish run on openml
run.publish()

```

---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-53-e1498e67825a> in <module>
      4 task = openml.tasks.get_task(3)
      5 # run model on task
----> 6 run = openml.runs.run_model_on_task(clf, task)
      7 # publish run on openml
      8 run.publish()

/miniconda3/lib/python3.7/site-packages/openml/runs/functions.py in run_model_on_task(model, task, avoid_duplicate_runs, flow_tags, seed, add_local_measures, upload_flow, return_flow)
    104         seed=seed,
    105         add_local_measures=add_local_measures,
--> 106         upload_flow=upload_flow,
    107     )
    108     if return_flow:

/miniconda3/lib/python3.7/site-packages/openml/runs/functions.py in run_flow_on_task(flow, task, avoid_duplicate_runs, flow_tags, seed, add_local_measures, upload_flow)
    172         task, flow = flow, task
    173 
--> 174     flow.model = flow.extension.seed_model(flow.model, seed=seed)
    175 
    176     # We only need to sync with the server right now if we want to upload the flow,

AttributeError: 'NoneType' object has no attribute 'seed_model'

You mentioned that this should  be fixed via #722, but I am still encountering the same error. 

@amueller

### PR DESCRIPTION
**Pull request by amueller**
closes #734.

### CODE REVIEW

**Comment by mfeurer** [comment]
Yep, that should solve it, thanks for the PR. However, it seems like you forgot to add the `MyLR` class.

**Comment by amueller** [comment]
damn, should be MyDummy now... I'll fix after lunch.

File: openml/flows/flow.py [hunk 1]
```diff
@@ -131,8 +131,10 @@ def __init__(self, name, description, model, components, parameters,
         self.language = language
         self.dependencies = dependencies
         self.flow_id = flow_id
-
-        self._extension = get_extension_by_flow(self)
+        if extension is None:
```
Comment (amueller) [inline openml/flows/flow.py]: Do you want me to remove this? Would work without it but I found it more explicit to directly give the extension.

**Comment by amueller** [comment]
probably fine now ;)

File: openml/flows/flow.py (hunk 1 above)
Comment (mfeurer) [reply openml/flows/flow.py]: This is totally fine by me.

**Comment by amueller** [comment]
I think the failures now are unrelated?

**Comment by amueller** [comment]
ping @PGijsbers ;)
This one is what prevents us from pushing flows with non-sklearn components

**Pull request merged**